b2/console_tool.py

```diff
@@ -355,6 +355,100 @@ def run(self, args):
         return 0
 
 
+class CopyFile(Command):
+    """
+    b2 copy-file [--metadataDirective [copy|replace]] [--contentType <contentType>] \\
```

Comment:
This does not yet work for files over 5GB, right?  I don't see any code to handle large files, parts, etc.

Reply:
that's up to b2sdk, but yes, sdk 0.1.8 and sdk 0.1.10 can only handle small files. The PRs were 300 LOC each already, we wanted to merge that first before adding large file support.

Reply:
I think it's worth documenting the limitation here.  Users of the CLI may not read the SDK docs.